code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/process/ServicesReconcilePostTrigger.java

```diff
@@ -0,0 +1,52 @@
+package io.cattle.platform.servicediscovery.process;
+
+import io.cattle.platform.archaius.util.ArchaiusUtil;
+import io.cattle.platform.core.model.Service;
+import io.cattle.platform.engine.handler.HandlerResult;
+import io.cattle.platform.engine.handler.ProcessPostListener;
+import io.cattle.platform.engine.process.ProcessInstance;
+import io.cattle.platform.engine.process.ProcessState;
+import io.cattle.platform.process.common.handler.AbstractObjectProcessLogic;
+import io.cattle.platform.servicediscovery.deployment.DeploymentManager;
+import io.cattle.platform.servicediscovery.service.ServiceLookup;
+
+import java.util.Collection;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import com.netflix.config.DynamicStringListProperty;
+
+@Named
+public class ServicesReconcilePostTrigger extends AbstractObjectProcessLogic implements ProcessPostListener {
+    private static final DynamicStringListProperty PROCESS_NAMES = ArchaiusUtil
+            .getList("service.reconcile.posttrigger.processes");
```

Comment:
Yeah.  I remember seeing it for the config updates too.
I like having the ServicesReconcilePostTrigger class.  But I think I prefer having them all explicitly listed in the getProcessNames() function as ugly as it may seem.  I guess I still don't see much value making it configurable via an environment variable and I think having them explicitly listed in getProcessNames() makes it easier for someone to debug the system.